content/applications/finance/fiscal_localizations/india.rst

```diff
@@ -348,6 +351,33 @@ fill out the :guilabel:`Cancel reason` and :guilabel:`Cancel remarks` fields. Th
      the government portal. You can click :guilabel:`Process Now` if you want to process the invoice
      immediately.
 
+.. _india/gstin_status:
+
+Indian Check GSTIN Status
+=========================
+
+The :guilabel:`Indian - Check GST Number Status` module allows you to verify the status of a GSTIN
+(Goods and Services Tax Identification Number) directly from Odoo.
+
+There are two ways to verify the status of a GST Number in Odoo:
+
+#. To verify the status of a partner's GST Number, go to :menuselection:`Accounting/Invoicing --> Customers/Vendors`
+   and select a partner. In the :guilabel:`GSTIN` field, click on the :guilabel:`Check GSTIN Status` button.
+
+   .. image:: india/gstin-status-partner.png
+      :alt: Check GSTIN status of a partner
+
+#. To verify the status of a GST Number entered in an invoice/bill, go to :menuselection:`Accounting/Invoicing --> Invoices/Vendor Bills`
+   and create a new invoice/bill and post the invoice/bill. In the :guilabel:`GST Status` field, click on the :guilabel:`GSTIN Status Refresh` button.
+
+   .. image:: india/gstin-status-invoice.png
+      :alt: Check GSTIN status of an invoice
+
+A notification will be popped as shown below and the partner GSTIN status and verification date will be logged in partner form.
+
+.. image:: india/gstin-status-notification.png
+   :alt: GSTIN status notification
```

Comment:
If I understand correctly, this section only applies to contacts? I suggest moving it after that line and updating the text as follows, which would allow us to remove the screenshot:
```suggestion
A notification is displayed to confirm the status update and the GSTIN status and verification date are logged in the contact's chatter.
```

Reply:
It doesn't matter from where the user clicks the button but in case of invoice/bill. It clearly says that we only post the log note in contact form (to keep the record in case the GST number deactivates) and a normal notification appears(from both the form)